### PR TITLE
TLDR-599 re-label datasets

### PR DESCRIPTION
**Pull request by NastyBoget**

### CODE REVIEW

File: labeling/train_dataset/api/async_archive_handler.py [hunk 1]
```diff
@@ -45,7 +45,12 @@ def _handle_archive(self, uid: str, path: str, parameters: dict) -> str:
                     self.__handle_one_file(archive, file, parameters)
                     self.progress[uid] = f"files done\t= {i + 1} \n files_in_progress\t= {0}\n total\t= {len(archive.namelist())}"
 
-            task, _ = self.tasker.create_tasks(type_of_task=parameters["type_of_task"], task_size=int(parameters["task_size"]), task_uid=uid)
+            task, _ = self.tasker.create_tasks(
```
Comment (oksidgy) [inline labeling/train_dataset/api/async_archive_handler.py:48]: я не очень понимаю, зачем здесь был код запуска асинхронной создания тасков, когда все равно это на одной машине делается. Код сложно читается из-за этого

File: labeling/train_dataset/taskers/concrete_taskers/abstract_line_label_tasker.py [hunk 2]
```diff
@@ -103,7 +103,7 @@ def _create_one_task(self, task: List[List[dict]], task_id: int, job_uid: str, *
         task_items = OrderedDict()
```
Comment (oksidgy) [inline labeling/train_dataset/taskers/concrete_taskers/abstract_line_label_tasker.py:103]: надо потом выпилить будет TableTasker из-за него два абстрактных класса таскеров, вместо одного

Comment (oksidgy) [reply labeling/train_dataset/taskers/concrete_taskers/abstract_line_label_tasker.py:103]: и убрать header-footer tasker тоже потом надо

File: labeling/train_dataset/api/async_archive_handler.py (hunk 1 above)
Comment (NastyBoget) [reply labeling/train_dataset/api/async_archive_handler.py:48]: Создала [TLDR-637](https://jira.intra.ispras.ru/browse/TLDR-637)

File: labeling/train_dataset/taskers/concrete_taskers/abstract_line_label_tasker.py (hunk 2 above)
Comment (NastyBoget) [reply labeling/train_dataset/taskers/concrete_taskers/abstract_line_label_tasker.py:103]: Создала [TLDR-638](https://jira.intra.ispras.ru/browse/TLDR-638)
